docs/data_formats.md

```diff
@@ -207,7 +207,7 @@ This is an example:
   "scale" : [ 11.24, 11.24, 28 ]
 }
 ```
-
+TODO: terminology resolution vs magnification
```

Comment:
I would leave the paragraph as is (i.e., don't change "resolution"), but add a sentence explaining that we also use the term "magnifications" synonymously in the UI.